packages/cli/src/util/logs.ts

```diff
@@ -257,3 +243,69 @@ function getSourceIcon(source: string) {
   if (source === 'serverless') return 'ƒ';
   return ' ';
 }
+
+function sanitize(log: any): string {
+  return (
+    (log.text || '')
+      .replace(/\n$/, '')
+      .replace(/^\n/, '')
+      // eslint-disable-next-line no-control-regex
+      .replace(/\x1b\[1000D/g, '')
+      .replace(/\x1b\[0K/g, '')
+      .replace(/\x1b\[1A/g, '')
+  );
+}
+
+function colorize(text: string, log: any) {
+  if (isError(text, log)) {
+    return chalk.red(text);
+  }
+  return isWarning(text) ? chalk.yellow(text) : text;
+}
+
+function isError(text: string, log: any) {
+  return (
+    (log.type === 'error' && !ignoreStdErr(text)) ||
```

Comment:
Since we haven't enabled that flag for customers yet, let's maybe not use it for `vc logs` either?

Reply:
Good point! 
This simplifies the logic a lot, because we don't have `ignoreStdErr` at all.